style.css

```diff
@@ -0,0 +1,112 @@
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap");
+
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  background-color: #170606;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
```

Comment:
- [ ] [REQUIRED]  Please insert the desired image into the body, just like [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471). I have highlighted them in the picture below!

![image](https://user-images.githubusercontent.com/63915024/185959522-b5d9d83e-0fbd-42be-80e9-72c60f918d59.png)

Reply:
- [ ] As the previous reviewer mentioned, please insert the required background image as seen in the Figma design. Your design should look similar to that of the Figma template.